#!/usr/bin/env python

PACKAGE='auv_vision'

from dynamic_reconfigure.parameter_generator_catkin import *
from dynamic_reconfigure.msg import SensorLevels

gen = ParameterGenerator()

gen.add( "feature_distance", int_t, SensorLevels.RECONFIGURE_RUNNING, "Min. euclidian distance between features", 20, 1, 100 )
gen.add( "feature_quality", double_t, SensorLevels.RECONFIGURE_RUNNING, "Fraction of feature quality measure below which features are rejected.", 0.1, 0.01, 1.0 )

exit(gen.generate(PACKAGE, "optical_flow", "OpticalFlow"))
